### PR TITLE
Fix - Issue #755 - Adds Acrostic heading support for \qa tags

### PR DESCRIPTION
**Pull request by richmahn**

### CODE REVIEW

File: usfm_tools/support/singlehtmlRenderer.py [hunk 1]
```diff
@@ -262,7 +262,7 @@ def renderFP(self, token):
         self.write(u'<br />')
 
     def renderQSS(self, token):
-        self.write(u'<i>')
+        self.write(u'<i style="float:right;">')
```
Comment (PhotoNomad0) [inline usfm_tools/support/singlehtmlRenderer.py]: @richmahn - will this work for RTL languages?

Comment (richmahn) [reply usfm_tools/support/singlehtmlRenderer.py]: @PhotoNomad0 Good question. You know more than me. Does floating right on a page where language is going right to left float left?

Comment (richmahn) [reply usfm_tools/support/singlehtmlRenderer.py]: I should at least class this <i> so that one could override this style with a style sheet.

Comment (richmahn) [reply usfm_tools/support/singlehtmlRenderer.py]: I was just trying to get it to be like this: https://ubsicap.github.io/usfm/poetry/index.html#qs-qs

**Review by PhotoNomad0 — APPROVED** [review]

**Review by bspidel — COMMENTED** [review]
do we have a convention of writing css into html tags? If not these 2 instances should be moved to css file.

**Comment by richmahn** [comment]
@bspidel USFM-Tools doesn't generate a CSS file. If you're talking about on door43.org, we could do our own changes if we want, but for out-of-the-box raw HTML that our USFM-Tools generates, we also need to provide styling, not just a class.

**Pull request merged**